client/mig-console/loader.go

```diff
@@ -110,13 +110,19 @@ r               refresh the loader entry (get latest version from database)
 			}
 			fmt.Printf("%v\n", string(jsonle))
 		case "key":
+			fmt.Println("New key component must be 32 alphanumeric characters long, or type 'generate' to generate one")
```

Comment:
I don't have an opinion on whether the length should be hardcoded or use constants, but you should use one or the other consistently, not both.